### ISSUE TITLE
Create object for Distance and for Speed (like Instant)

### ISSUE THREAD

**Issue by dennisguse** [OpenTracksApp/OpenTracks#706]
At a lot of places in the code base, we do some computation/formatting for speed/distance (later also includes elevation) like metric vs. miles or converting it.

It would be very helpful to put this into separate objects.

Blocked by #703  (if work in progress)

**Comment by dennisguse** [comment]
Duplicate of #679....
